deps/build.jl

```diff
@@ -51,7 +51,7 @@ provides(BuildProcess,
 )
 
 
-@osx_only begin
+@compat @static if is_apple()
```

Comment:
And being even more picky, since this is just a build script there's not really a performance benefit to have it evaluated at parse time, so the `@static` could be left out altogether.